lib/driver.js

```diff
@@ -792,18 +800,26 @@ class XCUITestDriver extends BaseDriver {
         );
       }
       let retryCount = 0;
+
       await retryInterval(startupRetries, startupRetryInterval, async () => {
         this.logEvent('wdaStartAttempted');
         if (retryCount > 0) {
           this.log.info(`Retrying WDA startup (${retryCount + 1} of ${startupRetries})`);
         }
         try {
-          // on xcode 10 installd will often try to access the app from its staging
+          if (usePreinstalledWDA) {
+            // Stop the existing process before starting a new one to start a fresh WDA process every session.
+            await this.mobileKillApp(this.wda.bundleIdForXctest);
+
+            // TODO: Define `prebuiltWDAPath` or something as a new capability.
+            // Then, we can start installing the WDA package to start a session without xcodebuild.
```

Comment:
I'll do this idea as another PR. Then, maybe we can guide users to prepare a wda package and use it (without preinstalling it as well)